lib/portlayer/network/endpoint.go

```diff
@@ -31,14 +31,15 @@ type alias struct {
 var badAlias = alias{}
 
 type Endpoint struct {
-	container *Container
-	scope     *Scope
-	ip        net.IP
-	gateway   net.IP
-	subnet    net.IPNet
-	static    bool
-	ports     map[Port]interface{} // exposed ports
-	aliases   map[string][]alias
+	container   *Container
+	scope       *Scope
+	ip          net.IP
+	gateway     net.IP
+	subnet      net.IPNet
+	static      bool
+	ports       map[Port]interface{} // exposed ports
```

Comment:
This map is still used to check exposed ports, to see if port is already consumed by other mapping.
The new attribute below is just for persistence. I don't think portlayer should care too much about the conversion from nat.Portbinding to string.